mangaki/mangaki/templates/fix/fix_suggestion.html

```diff
@@ -0,0 +1,143 @@
+{% extends "base.html" %}
+{% load staticfiles %}
+{% load posters %}
+
+{% block title %}Suggestion #{{ suggestion.pk }} - Mangaki{% endblock %}
+
+{% block body %}
+
+<h1>Suggestion #{{ suggestion.pk }}</h1><br/>
+
+{% if suggestion.is_checked %}
+<div class="alert alert-info">
+    <span class="glyphicon glyphicon-lock" aria-hidden="true"></span> Ce problème a été marqué comme résolu !
+</div>
+{% endif %}
+
+<div class="row">
+
+<div class="col-md-3" align="center">
+    {% include "mangaki/work_rating.html" with category=suggestion.work.category.slug work=suggestion.work simple=True %}
+</div>
+
+<div class="col-md-9">
+    <br/>
+    <div class="row">
+    <div class="col-md-push-1 col-sm-push-1 col-lg-10 col-sm-10">
+        <div class="input-group">
+            <span class="input-group-addon" id="basic-addon2">
+                <span class="glyphicon glyphicon-link"></span> Lien de partage
+            </span>
+            <input type="text" class="form-control" value="{{ request.build_absolute_uri }}">
+        </div>
+    </div>
+    </div>
+    <br/>
+
+    <div class="well">
+        <p>
+            <h4><strong>Problème&nbsp;:</strong>
+                {{ suggestion.get_problem_display }}
+            </h4>
+        </p>
+
+        <p>
+            <h5><strong>Envoyé par&nbsp;:</strong>
+                <a href="{% url "profile" suggestion.user %}">{{ suggestion.user }}</a>
+                <small>({{ suggestion.date }})</small>
+            </h5>
+        </p>
+
+        {% if suggestion.message %}
+        <hr/>
+        <p class="text-muted">{{ suggestion.message }}</p>
+        {% endif %}
+
+        {% for cluster, color in clusters %}
+            <hr/>
+            <p><strong>Cette œuvre est présente dans ce cluster&nbsp;:</strong></p>
+            <div class="cards-grid no-rating">
+                {% for work in cluster.works.all %}
+                    {% if work != suggestion.work %}
+                    {% include "mangaki/work_rating.html" with category=work.category.slug work=work %}
+                    {% endif %}
+                {% endfor %}
+            </div>
+            <hr/>
+            <p>
+                Ce cluster est <strong><span class="{{ color }}">{{ cluster.get_status_display.lower }}</span></strong>
+                {% if cluster.checker %} par <a href="{% url "profile" cluster.checker %}">{{ cluster.checker }}</a>{% endif %}.
+            </p>
+            {% if cluster.resulting_work %}
+            <p>
+                Ce cluster a résulté en l'œuvre
```

Comment:
A bit weird, we could say: "Ce cluster a été fusionné dans l'œuvre:" (but I'm not convinced also by my terminology… hahaha, if you have any ideas, don't hesitate!)